kratos/utilities/sensitivity_builder.h

```diff
@@ -79,6 +79,8 @@ class KRATOS_API(KRATOS_CORE) SensitivityBuilder
 
     void ClearSensitivities();
 
+    void Clear();
```

Comment:
can you please add function-descriptions to `Clear` and `ClearSensitivities`
It would be especially interesting to know the difference btw those methods

Reply:
@MFusseder if you do this, consider adding a function `ClearFlags()` and calling it together with `ClearSensitivities()` in `Clear()`.

Reply:
This is a good suggestion. I will add `ClearFlags`.

Reply:
The new function and a description is added .